packages/core/src/anchor/ethereum/__tests__/ethereum-anchor-service.retry-failure.test.ts

```diff
@@ -3,8 +3,8 @@ import { whenSubscriptionDone } from '../../../__tests__/when-subscription-done.
 import { generateFakeCarFile, FAKE_STREAM_ID, FAKE_TIP_CID } from './generateFakeCarFile.js'
 
 const MAX_FAILED_ATTEMPTS = 2
-const POLL_INTERVAL = 100 // ms
-const MAX_POLL_TIME = 500 // ms - to test if polling stops after this threshold
+const POLL_INTERVAL = 1000 // ms
```

Comment:
do we need to increase these values?  Isn't this just going to slow down the test?

Reply:
Changed back (increased for the demo)